tests/runfiles/linux.run

```diff
@@ -577,3 +577,9 @@ tests = ['zvol_swap_001_pos', 'zvol_swap_002_pos', 'zvol_swap_003_pos',
 
 [tests/functional/libzfs]
 tests = ['many_fds']
+
+[tests/functional/mmp]
+tests = ['mmp_online_active_import', 'mmp_online_inactive_import',
+    'mmp_thread_basic', 'mmp_interval_0', 'mmp_interval_negative',
+    'mmp_interval_0_import', 'mmp_interval_uberblock_writes']
+pre =
```

Comment:
Each test is calling `default_setup_noexit` and `default_cleanup_noexit`. The cleanup is there to be absolutely sure cleanup happens.